polkadot/xcm/xcm-builder/src/location_conversion.rs

```diff
@@ -317,6 +317,46 @@ impl<Network: Get<Option<NetworkId>>, AccountId: From<[u8; 32]> + Into<[u8; 32]>
 	}
 }
 
+/// Create a description of the remote treasury `location` if possible. No two locations should have
+/// the same descriptor.
+pub struct DescribeTreasuryVoice;
+
+impl DescribeLocation for DescribeTreasuryVoice {
+	fn describe_location(location: &MultiLocation) -> Option<Vec<u8>> {
+		Some(match location {
+			// Used on the relay chain for sending paras that use 32 byte accounts
+			MultiLocation {
+				parents: 1,
```

Comment:
You could leave this as a terminal treasury voice, `parents: 0`, and reuse `DescribeFamily`